addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
In the case of a line-wrapped comment, absolutely go ahead an include a header/footer - I guess in general I would just be a fan of less comments, more readable code